SpringKonstant/src/org/usfirst/frc6357/robotcode/subsystems/TFMini.java

```diff
@@ -171,7 +145,8 @@ public double getSignalQualityDegree()
     private void initilize()
     {
     	sensor.write(config, config.length);
-    	sensor.setTimeout(1/100);				// Sensor is 100Hz, so timeout is .001 seconds
+    	sensor.setReadBufferSize(PACKET_LENGTH);
```

Comment:
Might we want to make the read buffer longer than the packet? If we make it the length of the packet and fall behind in processing, we'll lose data.

Reply:
We don't want old packets to be read so having the buffer as small as possible seemed like a sensible thing to do. at 115kbps with a 9 byte packet and 100 packets per second, 99% of the time, the line is idle so I expect we should be safe to buffer no more than 1 packet. It's probably OK to double this just in case, though.